brainstormforce/astra
# AST-714 - Duplicate id of ast-hf-mobile-menu in W3C validator

### Description
- https://wp-astra.atlassian.net/browse/AST-714
- Duplicate ID ast-hf-mobile-menu.

### Screenshots
- https://share.bsf.io/P8uG6vOp

### Types of changes
- Bug fix

### How has this been tested?
- Add toggle button for desktop header builder
- Make sure you have offcanvas menu in Offcanvas section
- Check on frontend with W3C validator

### Checklist:
